docs/rules/no-unassigned-import.md

```diff
@@ -1,4 +1,6 @@
-# import/no-unassigned-import: Forbid unassigned imports
```

Comment:
preserve

Reply:
This line from the body might capture this already:

> This rule aims to remove modules with side-effects by reporting when a module is imported but not assigned.

Reply:
Per https://github.com/import-js/eslint-plugin-import/pull/2582#discussion_r1008920390, i think the exact description phrase text needs to appear in the doc somewhere.